### PR TITLE
add composition cache diagram and clarify mat flow diagrams

### LINKED ISSUE #87: DRE exchange graphs vs. material flow graphs
Figures 6 & 7 show the material flow under 2 different scenarios, but do not show how the possible DRE paths change under those  scenarios.  In particular, I expect that the repo is always willing to take spent_fuel in addition to waste, but just doesn't because the separations competes favorably for it.

I recommend either:
1. adding text to clarify why these figures don't show the spent_fuel->repo relationship
2. adding figures that show the possible relationships, including possibly a 0 kg flow along the spent_fuel -> repo paths

### PR DESCRIPTION
**Pull request by rwcarlsen**
closes #87 and #79.

### CODE REVIEW

File: fundamentals/impl.tex [hunk 1]
```diff
@@ -322,13 +322,13 @@ \subsubsection{Resources and Materials}
     \item "never" (currently implemented) globally turns off all decay.
         The Material decay function does nothing.
 
+    \item "lazy" (currently implemented) decays any material whenever its
```
Comment (gidden) [inline fundamentals/impl.tex]: can you make sure the context around this proposed change is also updated?

Comment (rwcarlsen) [inline fundamentals/impl.tex]: I'm not sure what you are asking - could you clarify?

Comment (gidden) [inline fundamentals/impl.tex]: e.g., in the previous sentence: "C YCLUS can currently operate in 2 decay
modes, with 2 additional
modes likely to be added in future releases". Can you fix that and make
sure there aren't others like it, please.

On Mon, Feb 23, 2015 at 1:16 PM, Robert Carlsen <notifications@github.com>
wrote:

> In fundamentals/impl.tex
> <https://github.com/cyclus/fundamentals-paper/pull/97#discussion_r25192135>
> :
>
> > @@ -322,13 +322,13 @@ \subsubsection{Resources and Materials}
> >      \item "never" (currently implemented) globally turns off all decay.
> >          The Material decay function does nothing.
> >
> > +    \item "lazy" (currently implemented) decays any material whenever its
>
> I'm not sure what you are asking - could you clarify?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/fundamentals-paper/pull/97/files#r25192135>.
>



-- 
Matthew Gidden
Ph.D. Candidate, Nuclear Engineering
The University of Wisconsin -- Madison
Ph. 225.892.3192

Comment (rwcarlsen) [inline fundamentals/impl.tex]: got it - done.

**Comment by katyhuff** [comment]
This is great! Thanks so much @rwcarlsen !

**Pull request merged**